### DISCUSSION TITLE
Facebook RSS feed without login?

### DISCUSSION THREAD

**Discussion by markcellus** [snarfed/granary#599] [Q&A] [answered]
Hey @snarfed thanks for creating this package. I didn't know about it until interacting with you on blue sky thread! I see that you can use  it with Facebook but it requires a login. Will it eventually allow atom/rss Facebook feeds without login? Thanks again

**Comment by snarfed** [comment]
Hi @markcellus! Tentatively moved this to the granary repo, I think that's the project you meant. Sadly Facebook (and Instagram etc) API access has degraded a lot over time. We had to switch from using their API to scraping, and even that is poor at best. Background (focused on Bridgy, not granary, but problems are the same): https://brid.gy/about#rip-facebook

**Reply by markcellus** [reply, marked as answer]
Oh I see. I took a look at that link. So it sounds like there's no current way to get my Facebook profile data without a login. Thank you for the information.

**Comment by Charlotte-br560** [comment]
Hi there! Thanks for your interest in the package. Currently, accessing Facebook RSS feeds without login isn't supported due to Facebook's privacy policies. However, you should find an alternative solution: [Crawlbase](https://crawlbase.com/). It offers a range of features for accessing and parsing web content, including social media feeds. Hope that helps!

**Reply by snarfed** [reply]
^ spam, reported.
